config/locales/fr.yml

```diff
@@ -615,7 +618,7 @@ fr:
       new:
         add_member_to_group: Add member to group
         description: You are adding a new member to group %{name}
-        empty_state: Empty state
+        empty_state: There are no users with the provided email or id
```

Comment:
For the members select2 empty state. this should read `There are no users with the provided email or name` since we can only find the users with those attributes. Same applies to the project members. These changes would also apply to the `en.yml` file

Reply:
Fixed in 067de925a4b088475bd826e1d7b47f6cc26eed3b

en.yml file was already good.